github/util.py

```diff
@@ -42,6 +42,12 @@
 from model.github import GithubConfig
 
 
+# The limit is not documented explicitly in the GitHub docs. To see it, the error returned by
+# GitHub when creating a release with more then the allowed number of characters must be
+# looked at.
+MAXIMUM_GITHUB_RELEASE_BODY_LENGTH = 25000
```

Comment:
nits:
- rather use a docstr ('''.....''')?
- at least in comment mention unit (is it actually octets, or is it rather characters (not necessarily identical e.g. w/ utf-8)?)
- if it is just used _once_, preferrably either make it "private" (+ possibly lc), or inline it?